resources/auth/controller.js

```diff
@@ -129,3 +129,17 @@ exports.viewProfile = async (req, res) => {
     res.status(500).json({ message: `Error loading profile ${error.message}` });
   }
 };
+
+exports.storeImgUrl = async (req, res) => {
+  try {
+    // const { id } = req;
+    // const { secure_url } = req.photos.info;
+    await model.updateImageUrl(6, 'yet another Test123');
+    res.status(200).json({
+      // id: id,
+      // imgUrl: secure_url,
+    });
```

Comment:
Do you want to take these comments out, Joba?

Reply:
Also looks like the id is hardcoded?

Reply:
Yeah!

Issue addressed